From 26e8ec86695efd53d39bc54b7f8b9d7c5ab4111c Mon Sep 17 00:00:00 2001
From: Chris Cromer <chris@cromer.cl>
Date: Thu, 22 Jun 2017 19:52:04 -0400
Subject: [PATCH] make agetty quiet

---
 conf.d/agetty    | 3 +++
 init.d/agetty.in | 1 +
 2 files changed, 4 insertions(+)

diff --git a/conf.d/agetty b/conf.d/agetty
index 527ce4c2..5cfb5817 100644
--- a/conf.d/agetty
+++ b/conf.d/agetty
@@ -6,3 +6,6 @@
 
 # extra options to pass to agetty for this port
 #agetty_options=""
+
+# make agetty quiet
+#quiet="no"
diff --git a/init.d/agetty.in b/init.d/agetty.in
index f6cc12da..390b1317 100644
--- a/init.d/agetty.in
+++ b/init.d/agetty.in
@@ -16,6 +16,7 @@ term_type="${term_type:-linux}"
 command=/sbin/agetty
 command_args_foreground="${agetty_options} ${port} ${baud} ${term_type}"
 pidfile="/run/${RC_SVCNAME}.pid"
+export EINFO_QUIET="${quiet:-yes}"
 
 depend() {
 	after local
