--- Makefile.orig	2017-10-13 12:48:50.808988127 +0200
+++ Makefile	2017-10-13 12:55:11.727103625 +0200
@@ -93,7 +93,7 @@
 ACLOCAL_M4 = $(top_srcdir)/aclocal.m4
 am__aclocal_m4_deps = $(top_srcdir)/m4/attributes.m4 \
 	$(top_srcdir)/m4/ax_cxx_compile_stdcxx.m4 \
-	$(top_srcdir)/m4/gtk-doc.m4 $(top_srcdir)/m4/libtool.m4 \
+	$(top_srcdir)/m4/libtool.m4 \
 	$(top_srcdir)/m4/ltoptions.m4 $(top_srcdir)/m4/ltsugar.m4 \
 	$(top_srcdir)/m4/ltversion.m4 $(top_srcdir)/m4/lt~obsolete.m4 \
 	$(top_srcdir)/configure.ac
@@ -122,12 +122,9 @@
 SOURCES =
 DIST_SOURCES =
 RECURSIVE_TARGETS = all-recursive check-recursive cscopelist-recursive \
-	ctags-recursive dvi-recursive html-recursive info-recursive \
-	install-data-recursive install-dvi-recursive \
-	install-exec-recursive install-html-recursive \
-	install-info-recursive install-pdf-recursive \
-	install-ps-recursive install-recursive installcheck-recursive \
-	installdirs-recursive pdf-recursive ps-recursive \
+	ctags-recursive install-data-recursive \
+	install-exec-recursive \
+	install-recursive installcheck-recursive \
 	tags-recursive uninstall-recursive
 am__can_run_installinfo = \
   case $$AM_UPDATE_INFO_DIR in \
@@ -192,7 +189,7 @@
 ETAGS = etags
 CTAGS = ctags
 CSCOPE = cscope
-DIST_SUBDIRS = src bindings po doc perf glade
+DIST_SUBDIRS = src bindings po perf glade
 am__DIST_COMMON = $(srcdir)/Makefile.in $(srcdir)/config.h.in \
 	$(srcdir)/vte.pc.in $(top_srcdir)/src/vte/vteversion.h.in \
 	AUTHORS COPYING ChangeLog NEWS README compile config.guess \
@@ -292,12 +289,6 @@
 GOBJECT_LIBS = -lgobject-2.0 -lglib-2.0 
 GPERF = /usr/bin/gperf
 GREP = /bin/grep
-GTKDOC_CHECK = gtkdoc-check.test
-GTKDOC_CHECK_PATH = /usr/bin/gtkdoc-check
-GTKDOC_DEPS_CFLAGS = -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include 
-GTKDOC_DEPS_LIBS = -lgobject-2.0 -lglib-2.0 
-GTKDOC_MKPDF = /usr/bin/gtkdoc-mkpdf
-GTKDOC_REBASE = /usr/bin/gtkdoc-rebase
 GTK_3_16_CFLAGS = -pthread -I/usr/include/gtk-3.0 -I/usr/include/at-spi2-atk/2.0 -I/usr/include/at-spi-2.0 -I/usr/include/dbus-1.0 -I/usr/lib64/dbus-1.0/include -I/usr/include/gtk-3.0 -I/usr/include/gio-unix-2.0/ -I/usr/include/cairo -I/usr/include/pango-1.0 -I/usr/include/atk-1.0 -I/usr/include/cairo -I/usr/include/pixman-1 -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/freetype2 -I/usr/include/harfbuzz -I/usr/include/libpng16 -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/libpng16 -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include 
 GTK_3_16_LIBS = -lgtk-3 -lgdk-3 -lpangocairo-1.0 -lpango-1.0 -latk-1.0 -lcairo-gobject -lcairo -lgdk_pixbuf-2.0 -lgio-2.0 -lgobject-2.0 -lglib-2.0 
 GTK_API_MAJOR_VERSION = 3
@@ -307,7 +298,6 @@
 GTK_LIBS = -lgtk-3 -lgdk-3 -lpangocairo-1.0 -lpango-1.0 -latk-1.0 -lcairo-gobject -lcairo -lgdk_pixbuf-2.0 -lgio-2.0 -lgobject-2.0 -lglib-2.0 
 GTK_PREFIX = /usr
 HAVE_CXX11 = 1
-HTML_DIR = ${datadir}/gtk-doc/html
 INSTALL = /usr/bin/install -c
 INSTALL_DATA = ${INSTALL} -m 644
 INSTALL_PROGRAM = ${INSTALL}
@@ -343,7 +333,6 @@
 LT_SYS_LIBRARY_PATH = 
 LT_VERSION_INFO = 4800:3:4800
 MAINT = 
-MAKEINFO = ${SHELL} /sources/xc/xfce4/vte-0.48.3/missing makeinfo
 MANIFEST_TOOL = :
 MKDIR_P = /bin/mkdir -p
 MKINSTALLDIRS = ./mkinstalldirs
@@ -418,17 +407,13 @@
 builddir = .
 datadir = ${datarootdir}
 datarootdir = ${prefix}/share
-docdir = ${datarootdir}/doc/${PACKAGE_TARNAME}
-dvidir = ${docdir}
 exec_prefix = ${prefix}
 host = x86_64-pc-linux-gnu
 host_alias = 
 host_cpu = x86_64
 host_os = linux-gnu
 host_vendor = pc
-htmldir = ${docdir}
 includedir = ${prefix}/include
-infodir = ${datarootdir}/info
 install_sh = ${SHELL} /sources/xc/xfce4/vte-0.48.3/install-sh
 intltool__v_merge_options_ = $(intltool__v_merge_options_$(AM_DEFAULT_VERBOSITY))
 intltool__v_merge_options_0 = -q
@@ -436,13 +421,10 @@
 libexecdir = ${exec_prefix}/libexec
 localedir = ${datarootdir}/locale
 localstatedir = ${prefix}/var
-mandir = ${datarootdir}/man
 mkdir_p = $(MKDIR_P)
 oldincludedir = /usr/include
-pdfdir = ${docdir}
 prefix = /usr
 program_transform_name = s,x,x,
-psdir = ${docdir}
 sbindir = ${exec_prefix}/sbin
 sharedstatedir = ${prefix}/com
 srcdir = .
@@ -453,7 +435,7 @@
 top_srcdir = .
 NULL = 
 ACLOCAL_AMFLAGS = -I m4
-SUBDIRS = src bindings po doc perf $(am__append_1)
+SUBDIRS = src bindings po perf $(am__append_1)
 EXTRA_DIST = \
 	HACKING \
 	autogen.sh \
@@ -464,9 +446,9 @@
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = vte-$(VTE_API_VERSION).pc
 DISTCHECK_CONFIGURE_FLAGS = \
-	--enable-gtk-doc \
+	--disable-gtk-doc \
 	--enable-introspection \
-	--disable-vala \
+	--enable-vala \
 	--disable-test-application \
 	--disable-silent-rules \
 	--with-gtk=$(GTK_API_VERSION)
@@ -907,44 +889,12 @@
 distclean-am: clean-am distclean-generic distclean-hdr \
 	distclean-libtool distclean-tags
 
-dvi: dvi-recursive
-
-dvi-am:
-
-html: html-recursive
-
-html-am:
-
-info: info-recursive
-
-info-am:
-
 install-data-am: install-pkgconfigDATA
 
-install-dvi: install-dvi-recursive
-
 install-dvi-am:
 
 install-exec-am:
 
-install-html: install-html-recursive
-
-install-html-am:
-
-install-info: install-info-recursive
-
-install-info-am:
-
-install-man:
-
-install-pdf: install-pdf-recursive
-
-install-pdf-am:
-
-install-ps: install-ps-recursive
-
-install-ps-am:
-
 installcheck-am:
 
 maintainer-clean: maintainer-clean-recursive
@@ -957,14 +907,6 @@
 
 mostlyclean-am: mostlyclean-generic mostlyclean-libtool
 
-pdf: pdf-recursive
-
-pdf-am:
-
-ps: ps-recursive
-
-ps-am:
-
 uninstall-am: uninstall-pkgconfigDATA
 
 .MAKE: $(am__recursive_targets) all install-am install-strip
@@ -975,15 +917,12 @@
 	dist-all dist-bzip2 dist-gzip dist-lzip dist-shar dist-tarZ \
 	dist-xz dist-zip distcheck distclean distclean-generic \
 	distclean-hdr distclean-libtool distclean-tags distcleancheck \
-	distdir distuninstallcheck dvi dvi-am html html-am info \
-	info-am install install-am install-data install-data-am \
-	install-dvi install-dvi-am install-exec install-exec-am \
-	install-html install-html-am install-info install-info-am \
-	install-man install-pdf install-pdf-am install-pkgconfigDATA \
-	install-ps install-ps-am install-strip installcheck \
+	distdir distuninstallcheck install install-am install-data \
+	install-data-am install-exec install-exec-am \
+	install-pkgconfigDATA install-strip installcheck \
 	installcheck-am installdirs installdirs-am maintainer-clean \
 	maintainer-clean-generic mostlyclean mostlyclean-generic \
-	mostlyclean-libtool pdf pdf-am ps ps-am tags tags-am uninstall \
+	mostlyclean-libtool tags tags-am uninstall \
 	uninstall-am uninstall-pkgconfigDATA
 
 .PRECIOUS: Makefile
